src/app.rs

```diff
@@ -253,19 +253,6 @@ where
     ///     .default_service(
     ///         web::route().to(|| HttpResponse::NotFound()));
     /// ```
-    ///
-    /// It is also possible to use static files as default service.
-    ///
-    /// ```
-    /// use actix_web::{web, App, HttpResponse};
-    ///
-    /// let app = App::new()
-    ///     .service(
-    ///         web::resource("/index.html").to(|| HttpResponse::Ok()))
-    ///     .default_service(
-    ///         web::to(|| HttpResponse::NotFound())
-    ///     );
-    /// ```
```

Comment:
This was weird - it was not actually using a static file for `default_service` as far as I can see.